examples/example_compiler_gym_service/__init__.py

```diff
@@ -8,46 +8,58 @@
 from compiler_gym.util.runfiles_path import runfiles_path
 
 
-class CodesizeReward(Reward):
-    """An example reward that uses changes in the "codesize" observation value
+class RuntimeReward(Reward):
```

Comment:
I wonder if this could be even more thin -- seems like the pertinent info could be passed into the Reward() constructor (deterministic, platform dependent, etc) and everything is just a scalar of some kind by default

Reply:
Good idea. This is kind of how the LLVM backend does it ([e.g.](https://github.com/facebookresearch/CompilerGym/blob/development/compiler_gym/envs/llvm/llvm_env.py#L84-L91)), but I haven't tried to generalize it yet. Filing #130 to follow up